core/tests/puppeteer-acceptance-tests/utilities/user/blog-post-editor.ts

```diff
@@ -145,22 +149,73 @@ export class BlogPostEditor extends BaseUser {
   }
 
   /**
+   * This is a composite function that can be used when a straightforward, simple blog published is required.
    * This function publishes a blog post with given title.
    */
   async publishNewBlogPostWithTitle(newBlogPostTitle: string): Promise<void> {
```

Comment:
I think it's better to rename it as publishNewBlogPost as it also includes a thumbnail, body, and tags.

Reply:
Done